saleor/graphql/product/mutations/products.py

```diff
@@ -934,23 +868,6 @@ def get_instance(cls, info, **data):
     @transaction.atomic
     def save(cls, info, instance, cleaned_input):
         instance.save()
-        if not instance.product_type.has_variants:
-            site_settings = info.context.site.settings
-            track_inventory = cleaned_input.get(
-                "track_inventory", site_settings.track_inventory_by_default
-            )
-            sku = cleaned_input.get("sku")
-            variant_price = cleaned_input.get("base_price")
-
-            variant = models.ProductVariant.objects.create(
-                product=instance,
-                track_inventory=track_inventory,
-                sku=sku,
-                price_amount=variant_price,
-            )
-            stocks = cleaned_input.get("stocks")
-            if stocks:
-                cls.create_variant_stocks(variant, stocks)
```

Comment:
Should we remove this function?

Reply:
If it's no longer needed then yes.

Reply:
This function is related with variants, if we want to separate creating products and creating variants, we also should remove this function.